cli/types/index.d.ts

```diff
@@ -910,6 +927,165 @@ declare namespace Cypress {
+     * @default null
+     */
+    baseUrl: string | null;
+    /**
+     * Any values to be set as [environment variables](https://docs.cypress.io/guides/guides/environment-variables.html)
+     * @default {}
+     */
+    env: { [key: string]: any };
+    /**
+     * A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests. Cypress uses minimatch with the options: {dot: true, matchBase: true}. We suggest using http://globtester.com to test what files would match.
+     * @default "*.hot-update.js"
+     */
+    ignoreTestFiles: string | string[];
+    /**
+     * The number of tests for which snapshots and command data are kept in memory. Reduce this number if you are experiencing high memory consumption in your browser during a test run.
+     * @default 50
+     */
+    numTestsKeptInMemory: number;
+    /**
+     * Port used to host Cypress. Normally this is a randomly generated port
+     * @default null
+     */
+    port: number | null;
+    /**
+     * The [reporter](https://docs.cypress.io/guides/guides/reporters.html) used when running headlessly or in CI
+     * @default "spec"
+     */
+    reporter: string;
+    /**
+     * Whether to take a screenshot on test failure when running headlessly or in CI
+     * @default true
+     */
+    screenshotOnHeadlessFailure: boolean;
+    /**
+     * Whether Cypress will watch and restart tests on test file changes
+     * @default true
+     */
+    watchForFileChanges: boolean;
+    /**
+     * Time, in milliseconds, to wait until most DOM based commands are considered timed out
+     * @default 4000
+     */
+    defaultCommandTimeout: number;
+    /**
+     * Time, in milliseconds, to wait for a system command to finish executing during a [cy.exec()](https://on.cypress.io/api/exec) command
+     * @default 60000
+     */
+    execTimeout: number;
+    /**
+     * Time, in milliseconds, to wait for page transition events or [cy.visit()](https://on.cypress.io/api/visit), [cy.go()](https://on.cypress.io/api/go), [cy.reload()](https://on.cypress.io/api/reload) commands to fire their page load events
+     * @default 60000
+     */
+    pageLoadTimeout: number;
+    /**
+     * Time, in milliseconds, to wait for an XHR request to go out in a [cy.wait()](https://on.cypress.io/api/wait) command
+     * @default 5000
+     */
+    requestTimeout: number;
+    /**
+     * Time, in milliseconds, to wait until a response in a [cy.request()](https://on.cypress.io/api/request), [cy.wait()](https://on.cypress.io/api/wait), [cy.fixture()](https://on.cypress.io/api/fixture), [cy.getCookie()](https://on.cypress.io/api/getcookie), [cy.getCookies()](https://on.cypress.io/api/getcookies), [cy.setCookie()](https://on.cypress.io/api/setcookie), [cy.clearCookie()](https://on.cypress.io/api/clearcookie), [cy.clearCookies()](https://on.cypress.io/api/clearcookies), and [cy.screenshot()](https://on.cypress.io/api/screenshot) commands

```

Comment:
you don't have to use `https://on.cypress.io/api/request` just `https://on.cypress.io/request` would be better!

Reply:
Sure thing.

There are a lot of other URLs with the extra `/api` in the URL. I can update those URLs as well.